SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialWorker.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialWorker.h"
+
+#include "SpatialConstants.h"
+
+#include "Engine/World.h"
+
+
+bool USpatialWorker::CanHaveAuthority(const AActor* Actor)
```

Comment:
do you reckon it's worth us extending this to blueprint / a general set of methods for checking netmode / auth beyond just `HasAuthority` / `IsServer`?